src/core/error.ml

```diff
@@ -43,6 +43,9 @@ let short_type ctx t =
 	Should be called for each complementary error message.
 *)
 let compl_msg s = "... " ^ s
+let rec compl_located_msg = function
```

Comment:
wtf is "compl"? I know it's there already, but that's a strange abbreviation.

Reply:
Complementary information or something like that? If you have a better idea I can update those on this PR